pom.xml

```diff
@@ -166,7 +167,7 @@
         </dependency>
     </dependencies>
     <build>
-        <finalName>${project.artifactId}-${project.version}-${maven.build.timestamp}</finalName>
```

Comment:
I'd like to leave it as is. If you want shorter name please rename it after build using your-OS packaging tools.